### DISCUSSION TITLE
A few issues regarding staking on Mainnet

### DISCUSSION THREAD

**Discussion by kwunyeung** [oasisprotocol/community-forum#14] [Mainnet]
There are currently a few critical issues as $ROSE investors and hodlers while staking on Mainnet.

### 1. Locked $ROSE can't be delegated
Investors invested in $ROSE as early supporters. However, the locked $ROSE can't be delegated while the inflation has been enabled since Mainnet. This has become a punishment to the investors as their locked $ROSE value are being diluted with the current high inflation. The longer they can't delegate, the more they are being punished.

### 2. There is no redelegation
Staked $ROSE can't be redelegated to other validators after they are being staked to a validator. If a delegator wants to do so, it has to reclaim escrow, wait for 2 weeks and add escrow to another validator. This will discourage delegators to manage their $ROSE and study more on the validators wisely as they would not like to lose the inflation rewards gain during the unlock period.

### 3. Rewards are staked automatically
I am not quite sure the reason behind this implementation but delegators should have their own choice to choose whether they would like to stake or move the rewards. This is removing the free will of the delegators.

How the core team think of the above issues? Is there any plan that will tackle these in the future?

**Comment by 1xCL** [comment]
I think @kwunyeung has great points. 

Regarding #1, we need to implement https://github.com/oasisprotocol/oasis-core/issues/2322 so that not only investors can avoid dilution but also the Foundation's accounts can be managed according to the release schedule at the protocol level. Technically if the Foundation decides to reclaim escrows now, we will have more than 5 billion ROSE in circulation in 14 days.

**Reply by kwunyeung** [reply]
I would also suggest the Foundation to have a plan to cover the lost of the early investors. The fund from the reclaim escrows can be used in this way.

**Comment by kostko** [comment]
Thank you very much for the feedback.

A few notes to point out from the perspective of the Foundation:

#### Locked ROSE can’t be staked
Thanks for this. This took us a bit of time to respond to because we did some additional brainstorming and planning to see if there was any feasible way to do this and do it in a time window that would be beneficial. Unfortunately the conclusion is that at this time and given the way that the network is designed, this would be a very difficult technical change to implement and would impact the network in a way that is not feasible to support.

One note however. From the Amber Network to the Mainnet, the Foundation actually proposed an adjustment in staking rewards (from 15% to 20% for the first 6 months) to further support community members who stake their unlocked tokens to contribute to the security of the network.This was proposed and approved by the community at the time of Mainnet Beta launch and again at Mainnet launch. While certainly this is not a direct solution for the issue you note, the intent was to alleviate the concern you raise.

Of course we encourage you to continue to stake the unlocked tokens you have and also run validators, get involved in running ParaTime nodes, and upcoming incentivized testnets as ways to continue to engage in the community and further support the network going forward.

#### There is no redelegation
The Foundation is focused on the security of the network first and foremost. Support for redelegation seems like a small thing on first sight, but it would add a layer of complexity to the staking part to implement on par with our network security standards.

That being said, if there is enough interest in the community, the Foundation is committed to working with the community to design a spec (ADR) and facilitating its implementation.

#### Rewards are staked automatically
Yeah, this is not ideal and if there is sufficient interest from the community, this is something we can work with community members to address in Upgrade 2 (Upgrade 1 to deploy very soon of course!).
